source/MapOutfitterPanel.cpp

```diff
@@ -218,6 +218,8 @@ void MapOutfitterPanel::DrawItems()
 					info += " of weapon space";
 				else if(space && -outfit->Get("engine capacity") == space)
 					info += " of engine space";
+				else if(space && -outfit->Get("afterburner slot") == space)
```

Comment:
idk what you're doing here but you can include this in the last condition with an or between this and the engine capacity thing

that said I suspect the text shouldnt be the same